include/boost/gil/extension/io/raw/read.hpp

```diff
@@ -0,0 +1,40 @@
+/*
+    Copyright 2013 Christian Henning
+    Use, modification and distribution are subject to the Boost Software License,
+    Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+    http://www.boost.org/LICENSE_1_0.txt).
+*/
+
+/*************************************************************************************************/
+
+#ifndef boost_gil_extension_io_raw_read_hpp_
+#define boost_gil_extension_io_raw_read_hpp_
+
+////////////////////////////////////////////////////////////////////////////////////////
+/// \file
+/// \brief
+/// \author Olivier Tournaire \n
```

Comment:
Side comment: Doesn't this `author` clash with the copyright header?

Reply:
I haven't looked at any file content.